code/game/objects/items/singularityhammer.dm

```diff
@@ -80,6 +89,7 @@
 			playsound(user, 'sound/weapons/marauder.ogg', 50, TRUE)
 			var/turf/target = get_turf(A)
 			vortex(target,user)
+			START_PROCESSING(SSobj, src)

```

Comment:
Use a timer instead of processing this. It's only used to recharge.

Reply:
Aight, that is kinda jank